src/Tizen.Wearable.CircularUI.Forms.Renderer/ContentButtonRenderer.cs

```diff
@@ -0,0 +1,156 @@
+            {
+                _box = new EBox(XForms.NativeParent)
+                {
+                    MinimumWidth = _defaultMinimumSize,
+                    MinimumHeight = _defaultMinimumSize
+                };
+                _box.SetLayoutCallback(OnLayout);
+
+                _button = new EButton(XForms.NativeParent);
+                _button.BackgroundColor = EColor.Transparent;
+                _button.SetPartColor("effect", EColor.Transparent);
+                _button.SetPartColor("effect_pressed", EColor.Transparent);
+                _button.Show();
+
+                _pressed = new SmartEvent(_button, "pressed");
+                _unpressed = new SmartEvent(_button, "unpressed");
+                _clicked = new SmartEvent(_button, "clicked");
+
+                _pressed.On += OnPressed;
+                _unpressed.On += OnReleased;
+                _clicked.On += OnClicked;
+
+                _box.PackEnd(_button);
+
+                SetNativeView(_box);
+            }
+
+            UpdateView();
+            base.OnElementChanged(e);
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _pressed.On -= OnPressed;
+                _unpressed.On -= OnReleased;
+                _clicked.On -= OnClicked;
+            }
+            base.Dispose(disposing);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == ContentButton.ContentProterty.PropertyName)
+            {
+                UpdateView();
+            }
+            base.OnElementPropertyChanged(sender, e);
+        }
+
+        protected override Xamarin.Forms.Size MinimumSize()
+        {
+            var size = base.MinimumSize();
+
+            if (_content != null)
+            {
+                if (_content is IMeasurable im)
+                {
+                    size = im.Measure(_box.MinimumWidth, _box.MinimumHeight).ToDP();
+                }
+            }
+
+            size.Width = (size.Width > _box.MinimumWidth) ? size.Width : _box.MinimumWidth;
+            size.Height = (size.Height > _box.MinimumHeight) ? size.Height : _box.MinimumHeight;
+
+            return size;
+        }
+
+        void OnPressed(object sender, EventArgs args)
+        {
+            Element?.SendPressed();
+        }
+
+        void OnReleased(object sender, EventArgs args)
+        {
+            Element?.SendReleased();
+        }
+
+        void OnClicked(object sender, EventArgs args)
+        {
+            Element?.SendClicked();
+        }
+
+        void OnLayout()
+        {
+            if (_content != null)
+            {
+                _content.Geometry = _box.Geometry;
+            }
+
+            _button.Geometry = _box.Geometry;
+            _button.RaiseTop();
+        }
+
+        void UpdateView()
+        {
+            if (Element.Content != null)
```

Comment:
No need to remove the existing content before adding a new view?

Reply:
Fixed